libraries/apollo-compiler/src/test/graphql/com/example/measurements

```diff
@@ -2,251 +2,11 @@
 // If you updated the codegen and test fixtures, you should commit this file too.
 
 Test:                                                                                      Total LOC:
-aggregate-all                                                                                  203612
-aggregate-kotlin-responseBased                                                                  65709
-aggregate-kotlin-operationBased                                                                 42225
+aggregate-all                                                                                    2928
+aggregate-kotlin-responseBased                                                                   1301
+aggregate-kotlin-operationBased                                                                     0
 aggregate-kotlin-compat                                                                             0
-aggregate-java-operationBased                                                                   95678
+aggregate-java-operationBased                                                                    1627
```

Comment:
You'll need to generate that file without `testFilter`:

```
updateTestFixtures=true ./gradlew :apollo-compiler:test --tests "com.apollographql.apollo.compiler.CodegenTest"
```

Reply:
Nevermind, looks like @BoD did it 👍

Reply:
Haha sorry wen too fast :)

Reply:
Turns out only the `.expected` files were updated but not `measurements` which @ebrattli just updated [here](https://github.com/apollographql/apollo-kotlin/pull/6116/commits/a6be63d3dc8e473703570c3be43676b0d94ed6c9) so it should be all good now ✅